utils.star

```diff
@@ -0,0 +1,15 @@
+def get_agent_config_artifact(plan, agent_config_json):
```

Comment:
Cool; good idea to keep the default behaviour for now.. but can be removed later.